src/main/java/world/jnc/invsync/util/serializer/PlayerSerializer.java

```diff
@@ -58,6 +60,9 @@
               enderChestSyncModule,
               gameModeSyncModule,
               experienceSyncModule,
+              // Sync the solcarrot data before syncing the player's current health, because the
+              // solcarrot sets the player's maxhealth.
+              solCarrotSyncModule,
```

Comment:
Did you test that? Like does it break stuff if it happens after syncing the health?

Reply:
Yes, I tested it. Here's what can happen if they run out of order:
1. The player has a current health of 40 and max health of 40 in the database.
2. The player.dat is blank, so the player starts off with 10 current health and 10 max health.
3. The 40 current health is restored, but it's above the max of 10, so it gets capped at 10.
4. The max health of 40 is restored, but the current health was already set to 10.